src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { getStatusBarHeight } from 'react-native-iphone-x-helper';
+import { Box, Inline, Inset } from '@/design-system';
+
+import { NavbarSvgIcon } from './NavbarSvgIcon';
+import { NavbarItem } from './NavbarItem';
+import { NavbarTextIcon } from './NavbarTextIcon';
+
+const statusBarHeight = getStatusBarHeight(true);
```

Comment:
won't this be coverd by safe area?

Reply:
I think it will and we should rather use this package instead of relying on it

Reply:
we should ban usage of `react-native-iphone-x-helper` - it's deprecated for many years since react-native-safe-area-context exists

Reply:
Can change! Just saw this was used in other places – didn't know we had multiple approaches.